src/trust_anchor_util.rs

```diff
@@ -16,6 +16,7 @@
 
 use {Error, TrustAnchor};
 use cert::{certificate_serial_number, Cert, EndEntityOrCA, parse_cert_internal};
+use verify_cert::parse_validity;
```

Comment:
Based on the existing code, it seems like `parse_validty` should go into `cert` instead of `verify_cert`, like `certificate_serial_number`.